CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.8.1dev
 * [Fix] Fix error that was incorrectly converted into a print message
-
+* [Fix] Modified histogram query to ensure histogram binning is done correctly (#751)
```

Comment:
please move this to the latest section (0.9.0) you might need to rebase

Reply:
It seems like the latest version is 0.8.1dev in the master branch's CHANGELOD.md. Should I create a new section for 0.9.0?

Reply:
my bad, i was looking at my development branch. no need to do anything to the changelog. just add comments to your code.